### ISSUE TITLE
/daily rewards

### ISSUE THREAD

**Issue by ecoemmeck** [ClayC90/ClaysToolKit#21]
make the ability for players to choose between skill points or in game currency

**Comment by ClayC90** [comment]
Shall this be skill points in the absence of currency in the world?

So that when the world has started generating the currency via the mint+treasury, the daily reward will start being a sum of the currnecy.

**Comment by ecoemmeck** [comment]
currently it is kinda random and the server will choose on its own most of the time its currency
what im thinking is if you add something like the /reward 0 or /reward 1 option 
so say /daily c for currency and /daily s for skill points

**Comment by ClayC90** [comment]
got it.

In absence of currency server will give only skillpoints

If currency has been created, you will have to choose between skillpoints and currency.

**Comment by ecoemmeck** [comment]
yep pretty much because some players would rather the skill points if they are cashed up
